### PR TITLE
Added new lints for multiple RDN attributes

### PR DESCRIPTION
**Pull request by zhengping12**

### CODE REVIEW

File: lints/lint_issuer_multiple_attr_in_rdn.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		}
+	}
+	return ResultStruct{Result: Pass}, nil
+}
+
+func init() {
+	RegisterLint(&Lint{
+		Name:          "w_issuer_multiple_attr_in_rdn",
+		Description:   "Certificates should not have multiple attributes in a single RDN in the subject name",
+		Providence:    "aswlabs certlint",
```
Comment (kumarde) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: Can you make these all awslabs in another PR next (asw is not a thing)

File: lints/lint_issuer_multiple_attr_in_rdn.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// lint_issuer_multiple_attr_in_rdn.go
+
```
Comment (kumarde) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: Can you add exactly what this lint is supposed to do?

File: lints/lint_issuer_multiple_attr_in_rdn.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+func (l *IssuerRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *IssuerRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
+	return true
+}
+
+func (l *IssuerRDNHasMultipleAttribute) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	var issuer pkix.RDNSequence
```
Comment (kumarde) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: If the RDNSequence is important and used in a number of lints we should parse it out in x509.go

File: lints/lint_issuer_multiple_attr_in_rdn.go (hunk 1 above)
Comment (zhengping12) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: Basically all I'm PRing are from Tab2, or awslabs certlint. Also do you know of any efficient way to find the real providence?

File: lints/lint_issuer_multiple_attr_in_rdn.go (hunk 2 above)
Comment (zhengping12) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: It's described in the description section :)

File: lints/lint_issuer_multiple_attr_in_rdn.go (hunk 3 above)
Comment (zhengping12) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: It's already parsed, but pkix.go discards any attribute beyond the first. We don't want to store anything beyond the first, because they are not adhering to the standard.

File: lints/lint_issuer_multiple_attr_in_rdn.go (hunk 1 above)
Comment (kumarde) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: Nah I was just referring to the typo. We need to at some point look through and see where the awslabs ones are coming from, which will probably just be parsing the BR or RFC.

File: lints/lint_issuer_multiple_attr_in_rdn.go (hunk 3 above)
Comment (kumarde) [inline lints/lint_issuer_multiple_attr_in_rdn.go]: So is it the case that anything after the first one is a violation? As it stands it looks like you're checking if each RDN has a length > 1, not the other way around.

**Comment by kumarde** [comment]
Closing. Handling in #59.
